lms/services/canvas_api/client.py

```diff
@@ -250,22 +260,90 @@ def list_files(self, course_id, sort="position"):
                     "lms_id": file["id"],
                     "name": file["display_name"],
                     "size": file["size"],
+                    "parent_lms_id": file["folder_id"],
```

Comment:
Maybe just `folder_id`? No need to change the name from what Canvas calls it?

```suggestion
                    "folder_id": file["folder_id"],
```

Reply:
This is the format expected on the FE